meilisearch/tests/index/create_index.rs

```diff
@@ -147,13 +147,13 @@ async fn test_create_multiple_indexes() {
     let index3 = server.index("test3");
     let index4 = server.index("test4");
 
-    index1.create(None).await;
-    index2.create(None).await;
-    index3.create(None).await;
+    let (task1, _) = index1.create(None).await;
+    let (task2, _) = index2.create(None).await;
+    let (task3, _) = index3.create(None).await;
```

Comment:
I cannot select the right place but it’s the same as above, we can use a shared server + 4 unique index

Reply:
Here you can still uses a shared server

Reply:
sorry, must have overlooked that one in my revision 🙈

Reply:
No problem, that’s perfect 👌